Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml

```diff
@@ -1027,37 +1244,210 @@ tasks:
+          left:
+            value:
+              complex:
+                root: inputs.Project
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -640,
+          "y": 1600
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "56":
+    id: "56"
+    taskid: e083a0be-b96d-48bf-8203-6c3f57fedd72
+    type: condition
+    task:
+      id: e083a0be-b96d-48bf-8203-6c3f57fedd72
+      version: -1
+      name: Is Container alert type
+      description: ''
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "11"
+      "yes":
+      - "37"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              complex:
+                root: PrismaCloudCompute.AuditAlert
+                accessor: type
+            iscontext: true
+          right:
+            value:
+              simple: container
+          ignorecase: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: incident
+                accessor: prismacloudcomputeimage
+            iscontext: true
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -337.5,
+          "y": 2120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "57":
+    id: "57"
+    taskid: 358db035-2569-4031-86b8-1a98b8c05f01
+    type: condition
+    task:
+      id: 358db035-2569-4031-86b8-1a98b8c05f01
+      version: -1
+      name: Is there an image name
       description: ''
+      type: condition
+      iscommand: false
+      brand: ""
     nexttasks:
       '#default#':
       - "6"
       "yes":
-      - "44"
+      - "2"
     separatecontext: false
     conditions:
     - label: "yes"
       condition:
       - - operator: isNotEmpty
           left:
             value:
-              simple: inputs.baseUrl
+              complex:
+                root: PrismaCloudCompute.AuditAlert
+                accessor: image
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": -20
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "58":
+    id: "58"
+    taskid: 896ee77a-73e2-4a9f-82d0-c9339d159830
+    type: condition
+    task:
+      id: 896ee77a-73e2-4a9f-82d0-c9339d159830
+      version: -1
+      name: Assert Image Name Exist
```

Comment:
I think it's a typo?
Maybe appropriate name would be: "Check if image scan report was found"